web/themes/custom/server_theme/templates/server-theme-element--person-card.html.twig

```diff
@@ -0,0 +1,37 @@
+<div class="max-w-sm min-h-[360px] pt-8 h-full w-full bg-white rounded-xl shadow-xl text-center  flex flex-col items-center  gap-7 ">
```

Comment:
This isn't aligned with the rest of the codebase - I'd expect to see composition and re-use of other existing/new elements.